nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.github/CONTRIBUTING.md

```diff
@@ -52,6 +53,76 @@ These tests are run both with the latest available version of `Nextflow` and als
 * Fix the bug, and bump version (X.Y.Z+1).
 * A PR should be made on `master` from patch to directly this particular bug.
 
-## Getting help
+## Getting Help
 
 For further information/help, please consult the [{{ cookiecutter.name }} documentation](https://nf-co.re/{{ cookiecutter.short_name }}/usage) and don't hesitate to get in touch on the nf-core Slack [#{{ cookiecutter.short_name }}](https://nfcore.slack.com/channels/{{ cookiecutter.short_name }}) channel ([join our Slack here](https://nf-co.re/join/slack)).
+
+## Pipeline Contribution Conventions
+
+To make the {{ cookiecutter.short_name }} code and processing logic more understandable for new contributors and to ensure quality, we semi-standardise the way the code and other contributions are written.
+
+### Adding a New Module
+
+If you wish to contribute a new module, please use the following coding standards:
```

Comment:
I think the use of the word "module" here is a bit confusing, given DSL2. I guess you just mean a new step in the pipeline?